docs/content/preview/contribute/docs/syntax-diagrams.md

```diff
@@ -182,63 +184,63 @@ modified:   content/preview/api/ysql/the-sql-language/statements/cmd_do.md
 
-### generated documentation source
+### Generated documentation source
 
 - **The set of files that the [_diagram generator_](#diagram-generator) produces**.
 
-All are located within the [_syntax resources directory_](#syntax-resources-directory) tree. The set minimally includes the [_grammar diagrams file_](#grammar-diagrams-file-here). It also includes all of the [_free-standing generated grammar-diagram pairs_](#free-standing-generated-grammar-diagram-pair) from this. These will be included within ordinary content `.md` files by URL reference from the [_diagram inclusion HTML_](#diagram-inclusion-HTML) located in the ordinary content `.md` file that wants the [_syntax diagram_](#syntax-diagram).
+All are located within the [_syntax resources directory_](#syntax-resources-directory) tree. The set minimally includes the [_grammar diagrams file_](#grammar-diagrams-file). It also includes all of the [_free-standing generated grammar-diagram pairs_](#free-standing-generated-grammar-diagram-pair) from this. These will be included within ordinary content `.md` files by URL reference from the [_diagram inclusion HTML_](#diagram-inclusion-HTML) located in the ordinary content `.md` file that wants the [_syntax diagram_](#syntax-diagram).
 
 Notice that there is no garbage collection scheme for unreferenced [_generated documentation source_](#generated-documentation-source) files. Content authors must do this task manually.
 
-### grammar diagrams file ([here](https://docs.yugabyte.com/preview/api/ysql/syntax_resources/grammar_diagrams/#abort))
+### Grammar [diagrams file](https://docs.yugabyte.com/preview/api/ysql/syntax_resources/grammar_diagrams/#abort)
 
 ```output
 <your path>/yugabyte-db/docs/content/preview/api/ysql/syntax_resources/grammar_diagrams.md
 ```
 
 This contains every [_syntax diagram_](#syntax-diagram) that is generated from all of the [_syntax rules_](#syntax-rule) that are found in the [_diagram definition file_](#diagram-definition-file).
 
-### generated grammar-diagram pair
+### Generated grammar-diagram pair
 
 - **The Markdown source that produce the result, in the human-readable documentation, that the user sees as a [_syntax diagram_](#syntax-diagram)**.
 
 Use this term when you want to focus in the fact that a [_syntax diagram_](#syntax-diagram) is presented for the user as a tabbed "Grammar" and "Diagram" pair.
 
-Always found in the [_grammar diagrams file_](#grammar-diagrams-file-here) (but, uniquely, not here as a tabbed pair). Optionally found as a [_free-standing generated grammar-diagram pair_](#free-standing-generated-grammar-diagram-pair).
+Always found in the [_grammar diagrams file_](#grammar-diagrams-file) (but, uniquely, not here as a tabbed pair). Optionally found as a [_free-standing generated grammar-diagram pair_](#free-standing-generated-grammar-diagram-pair).
```

Comment:
These grammar diagrams file links don't work because the heading includes a link, which is always a bad idea.

Reply:
Agreed, and fixed.